CodingChallenges/CC_123_ChaosGame_2/Processing/CC_123_ChaosGame_2/CC_123_ChaosGame_2.pde

```diff
@@ -0,0 +1,55 @@
+// The Chaos Game
+// Daniel Shiffman
+// Part 1: https://youtu.be/7gNzMtYo9n4
+// https://thecodingtrain.com/CodingChallenges/123.1-chaos-game
+// Part 2: https://youtu.be/A0NHGTggoOQ
+// https://thecodingtrain.com/CodingChallenges/123.2-chaos-game
+
+ArrayList<PVector> points = new ArrayList<PVector>();
```

Comment:
Don't feel strongly either way but would it make sense to just have a plain array here that is created in `setup()` since the number of points is fixed? I guess this is less flexible for a version where the points might be interactive?

Reply:
This makes totally sense. Thanks for this advice. I'm still learning 😀